src/main/java/org/web3j/console/Runner.java

```diff
@@ -48,7 +51,14 @@
     public static void main(String[] args) throws Exception {
         System.out.println(LOGO);
 
-        CliConfig config = CliConfig.getConfig(CliConfig.getWeb3jConfigPath().toFile());
+        CliConfig config = CliConfig.getConfig(CliConfig.getConfigPath().toFile());
+
+        if (Arrays.asList(args).contains("--telemetry")) {
+            new Telemetry(config, args).uploadAnalytics();
+        } else {
+            Telemetry.invokeAnalyticsUpload(args);
```

Comment:
What's the deal here? So someone passes in --telemetry and something is uploaded, but else something is also uploaded?

Reply:
I should refactor this. Basically invokeAnalyticsUpload spawns a background Web3j process which is invoked with `--telemetry`. Then the child process will see that it's been invoked with `--telemetry`, and perform the upload.

Reply:
Ah, right, I get it now